### PR TITLE
 first send to hub

### PR DESCRIPTION
**Pull request by samuel408**
hello

### CODE REVIEW

**Comment by neiltraugh** [comment]
This thing looks good to me

**Review by yulduzetta — APPROVED** [review]
GTG 🚢

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (neiltraugh) [inline index.html]: hella nice code

**Review by neiltraugh — APPROVED** [review]

**Comment by neiltraugh** [comment]
Hey. Thanks lol. Regarding the project, there are public data repositories
(government) from where we can retrieve data for apis in case we decide on
the Covid-19 topic and how in-depth we go. Possibly CDC, NIH, or NIBSC.
These data sets are large so it’ll possibly be tricky.

Neil

**Review by yulduzetta — APPROVED** [review]
love it! gtg

**Pull request merged**